VnE/Source/Extensions/ChemPack/voronoi.py

```
@@ -0,0 +1,418 @@
+        colors = [v_color, e_color, p_color]
+        lists = []
+        for i, p in enumerate(polyh):
+            if not np.all(np.isclose(np.array([[1,0,0,0], [0,1,0,0], [0,0,1,0], [0,0,0,1]]), p.symop, atol=0.0001)):
+                p = p.copyTo(np.linalg.inv(p.symop))
+                polyh[i] = p
+            lists.append(p.createList(atoms_poly_list, colors))
+        sum_poly = Polyhedra.mergePolyhedra(polyh)
+        ls = sum_poly.createList(sum_poly_list, colors, new=True)
+
+        TREE_MODEL.insertRow(TREE_MODEL.rowCount())
+        for l in lists:
```

Ruff (0.15.2)

[error] Ambiguous variable name: `l`

(E741)